src/metaswitch/crest/api/ping.py

```diff
@@ -52,8 +52,10 @@ def get(self):
 
         try:
             # Use a DeferredList rather than gatherResults to wait
-            # for all of the clients to fail or succeed.
-            yield defer.DeferredList(gets, consume_errors=True)
+            # for all of the clients to fail or succeed. Don't consume errors -
```

Comment:
If the "don't consume errors" stuff is important, maybe better to make it explicit in the code with `consume_errors=False`?